webhook/hook.go

```diff
@@ -42,15 +45,20 @@ func asJSON(w http.ResponseWriter, status int, message string) {
 }
 
 func checkBearerToken(r *http.Request, c config.Configuration) error {
-	tokenHeader := r.Header.Get("Authorization")
-	if tokenHeader == "" {
-		return fmt.Errorf("Request does not have Authorization header")
-	}
-	headerElems := strings.Split(tokenHeader, " ")
-	if len(headerElems) != 2 || (len(headerElems) > 0 && headerElems[0] != "Bearer") {
-		return fmt.Errorf("Malformed Authorization header")
+	tokenHeader := r.Header.Get(AuthorizationHeader)
```

Comment:
A constant got introduced for the header but for the query parameter in the next line, an inline string gets used. I can live with both but would like to have consistency.